packages/browser-wallet/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 -   Use gRPC-web instead of json-RPC.
 
+### Fixed
+
+-   `chainChanged` event is correctly propagated to all (not just whitelisted) dapps listening for events through the wallet API.
```

Comment:
Is this something that we actually want? 

The way I understand this change is that if I just have a site opened without ever connecting to the wallet I would be able to listen to these events. Is that correct?

What information do I get from this event? Nothing more than the chain genesis has changed?

Reply:
Yes, I think this should be up for discussion as well. The only information emitted with the event is the genesis hash of the chain the wallet has been set to use.

I couldn't come up with any scenarios where this could be used for anything malicious, but might be some sort of "invasion of privacy" anyway?

Reply:
We discussed this, and decided that this is fine as implemented now.